test/TestAppUtils/Themes/Generic.xaml

```diff
@@ -44,11 +44,17 @@
                                         <SymbolIcon Symbol="Back"/>
                                     </Button>
                                     <Button x:Name="ToggleThemeButton"
-                                        VerticalAlignment="Stretch"
+                                        VerticalAlignment="Stretch" Margin="0,0,4,0"
                                         Style="{ThemeResource AccentButtonStyle}"
                                         AutomationProperties.AutomationId="__ToggleThemeButton" 
                                         Content="Toggle theme">
                                     </Button>
+                                    <Button
+                                        x:Name="SwitchCIDimensionsButton"
```

Comment:
Should this be a toggle button or toggle switch  ?

Reply:
Yes togglebutton seems like a more reasonable type.